CONTRIBUTING.md

```diff
@@ -191,8 +190,7 @@ into the source repository.
   preoccupied with other PRs. If it happens that your PR receives no review for
   a long time, don't hesitate to gently solicit a review by commenting or
   by explicitly mentioning a maintainer that you know is knowledgeable in the
-  area of the PR. You can also advertise the PR on devel@riot-os.org mailing
-  list and ask for a review there.
+  area of the PR.
```

Comment:
Why not add a link to the forum here as well?

Reply:
Sounds good, I've reworded the removed sentence to mention the forum.